Documents/customAnimal.md

```diff
@@ -0,0 +1,55 @@
+# How To Make A Custom Animal
+
+  
+
+## In this example we will be configuring a dragon animal.
+
+  
+
+### Where To Configure:
+
+The first thing you need to know is all configurations of the animal are done in console.
+
+  
+
+### Naming The Animal:
+
+The second thing you have to do is give the animal a name. The name of the animal does not effect any part of the program, so feel free to name it whatever you like!
+
+For Example,
+```console
+Animal: dragon
+```
+
+  
+
+### Setting Up Speed:
+
+Thirdly, what you need to do is input the speed of the animal, the strength of the animal, and the weight of the animal all on one line separated by a space. For example,
+
+```console
+Input speed, strength, weight: 4 17 160
+```
+
+if you want to find out how to increase or decrease speed, please refer to speedAlgorithm.md
+
+### Setting Up Defense:
+
+Once you have the speed finished you now have to configure the defensive stats of the custom animal. This part is pretty simple. If you want your animal to take no damage, you set the defense to 0. If you want your animal to take normal amount of damage, set the defense value to 1. If you want the animal to take more damage, have the defensive value be greater than 1.
```

Comment:
... so a value between zero and 1 means "below normal damage" (more durable) and above 1 mean higher than normal damage (less durable) up to some max limit?

Reply:
Yes, the defense value is just multiplying the opposite animal attack value. So if you were to set the defense value to 100, then it would take 100x the regular damage.